### DISCUSSION TITLE
How to use breakpoints to change fontSize in theme.ts ?

### DISCUSSION THREAD

**Discussion by rodrigoeduardo** [chakra-ui/chakra-ui#4752] [Q&A] [answered]
Hello!

I've created my breakpoints values like this:
`const breakpoints = createBreakpoints({
  base: '0em',
  sm: '960px',
  md: '1300px',
  lg: '1550px',
  xl: '1900px',
})`

And then, in my theme.ts file, I want to change fontSize according to breakpoints values. For example: at 'sm' breakpoint I want 75% of default fontSize.

Something like this:
`
export const theme = extendTheme({
    breakpoints,
    styles: { global: { html: { fontSize: {{ sm: "75%" }} } } }
`
I know that won't work, but just for example.

Anyone can help me with this?

**Comment by primos63** [comment, marked as answer]
You're close! There is a 2xl breakpoint that you probably want to address. The `base` breakpoint is already `'0em'` so no need to specify that unless you really want to.

```javascript
const breakpoints = createBreakpoints({
  sm: "960px",
  md: "1300px",
  lg: "1550px",
  xl: "1900px",
  '2xl': ''
})

export const theme = extendTheme({
  breakpoints,
  styles: {
    global: {
      html: {
        fontSize: {
          base: '75%',
          sm: '85%',
          md: '95%',
          lg: '100%',
          xl: '125%'
        }
      }
    }
  },
}

**Reply by rodrigoeduardo** [reply]
Thanks a lot!!

**Reply by miketdonahue** [reply]
This should be in the docs!

**Reply by primos63** [reply]
@meebix  You could open an issue in the [docs repo](https://github.com/chakra-ui/chakra-ui-docs) to add the example.
